code/controllers/subsystem/ticker.dm

```diff
@@ -462,7 +462,7 @@ SUBSYSTEM_DEF(ticker)
 		var/mob/living = player.transfer_character()
 		if(living)
 			qdel(player)
-			living.notransform = TRUE
+			ADD_TRAIT(living, TRAIT_NO_TRANSFORM, REF(src))
```

Comment:
Should probably have a unique source, it's a fairly important(?) source of notransform so seeing it at a glance would be nice

Reply:
Holding a string ref to `SSTicker` isn't a unique enough source?